mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -227,34 +269,39 @@ def fit(self, X, y, custom_feature_names=None, groups=None, **fit_params):
         else:
             X_ = X
 
+        # shouldn't we use X_ now?
```

Comment:
Yes, as far as I can tell we should. Good catch.

Reply:
Thanks for confirming it :)